vision/localize_objects.rb

```diff
@@ -0,0 +1,91 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def localize_objects image_path:
+  # [START vision_localize_objects]
+  # image_path = "Path to local image file, eg. './image.png'"
+
+  require "google/cloud/vision"
+
+  vision = Google::Cloud::Vision::V1.new
```

Comment:
`Google::Cloud::Vision.new`

It defaults to `Google::Cloud::Vision.new version: :V1`

For Alpha/Beta versions, you can provide `version: :VERSION`

Reply:
with this change, the tests failed with the following error:
```
NoMethodError:
       undefined method `batch_annotate_images' for #<Google::Cloud::Vision::Project:0x00007feb084a4c50>
```

Reply:
https://github.com/googleapis/google-cloud-ruby/issues/2482

Accepted as a feature request

Reply:
PR sent and pending LGTM & merge & publish new gem
https://github.com/googleapis/google-cloud-ruby/pull/2487#pullrequestreview-159102063